app/constants.js

```diff
@@ -0,0 +1,9 @@
+export const MajorColorNames = [
+    "WHITE", "RED", "BLACK", "YELLOW", "VIOLET"
+];
+export const MinorColorNames = [
+	"BLUE", "ORANGE", "GREEN", "BROWN", "SLATE"
+];
+export const allColorPairs = MajorColorNames.flatMap(majorColor =>
```

Comment:
Nice use of `flatMap`. If we print this, we would have the extra requirement requested in the README too :)